pkg/kubelet/kubelet.go

```diff
@@ -140,6 +140,9 @@ const (
 	ContainerGCPeriod = time.Minute
 	// Period for performing image garbage collection.
 	ImageGCPeriod = 5 * time.Minute
+
+	// Maximum period to wait for pod volume setup operations
+	maxWaitForVolumeOps = 5 * time.Minute
```

Comment:
We'll probably want this to be longer like 20 minutes. See https://github.com/kubernetes/kubernetes/issues/15382#issuecomment-153268655